src/data/bliss_semantics.py

```diff
@@ -1,87 +1,227 @@
+    "24668": {"type": "POS", "value": "noun", "category": "grammatical", "features": {"gender": "feminine"}, "notes": "for teaching purposes", "equivalent_modifier": "14166", "priority":"2"}, # the female modifier is used more. Indicator is not used in communication
+    "24669": {"type": "POS", "value": "noun", "category": "grammatical", "features": {"person": "first-person", "notes": "for teaching purposes"},
+    "28043": {"type": "POS", "value": "verb", "category": "grammatical", "features": {"tense": "null", "voice": "null", "mood": "null", "aspect": "continuous", "form": "inflected"}, "notes": "for teaching purposes"},
+    "28044": {"type": "POS", "value": "noun", "category": "grammatical", "features": {"number": "plural", "definiteness": "definite"},
     "28045":
     {
         "and": [
-            {"type": "POS", "value": "noun", "category": "grammatical"},
-            {"type": "NUMBER", "value": "plural", "category": "grammatical"}
+            {"type": "POS", "value": "noun", "category": "grammatical", "features": {"definiteness": "definite"}},
+            {"type": "TYPE_SHIFT", "value": "concretization", "category": "semantic"},
         ]
     },
     "28046": {
         "and": [
-            {"type": "DEFINITENESS", "value": "definite", "category": "grammatical"},
-            {"type": "POS", "value": "noun", "category": "grammatical"},
-            {"type": "NUMBER", "value": "plural", "category": "grammatical"}
+            {"type": "POS", "value": "noun", "category": "grammatical", "features": {"number": "plural", "definiteness": "definite"}},
+            {"type": "TYPE_SHIFT", "value": "concretization", "category": "semantic"},
+        ]
+    },
+    
+    # indicator (adverb)
+    "24665": {"type": "POS", "value": "adverb", "category": "grammatical", "notes": "for teaching purposes", "priority": "2"},
+    # similar to ID: 8993; 
+    "24807": {"type": "POS", "value": "verb", "category": "grammatical", "features": {"tense": "present", "voice": "null", "mood": "declarative", "aspect" : "null", "form": "inflected"}, "notes": "for teaching purposes", "priority": "2"},
+    "25458": {"type": "POS", "value": "noun", "category": "grammatical", "features": {"size": "diminutive", "form": "inflected"}, "notes": "for teaching purposes", "equivalent_modifier": "28052", "priority":"2"}, # the diminutive modifier is used more. Indicator is not used
+
+    # added more indicators from WinBliss
+    # imperative mood
+    "24670": {"type": "POS", "value": "verb", "category": "grammatical", "features": {"tense": "null", "voice": "null", "mood": "imperative", "aspect" : "null", "form": "inflected"}},
+    # 3 participles
+    "24674": {"type": "POS", "value": "verb", "category": "grammatical", "features": {"tense": "past", "voice": "null", "mood": "null", "aspect" : "null", "form": {"inflected", "past-participle-1"}, "notes": "for teaching purposes"}, # mood and aspect may vary language to lanaguage on how its used
```

Comment:
The comment of `# mood and aspect may vary language to lanaguage on how its used` appears here and in the following two lines. If this comment applies broadly and not specific to these lines, it's better to write in the README section.

This is also a general comment applying to other locations.